src/components/ContactCard/ContactForm.jsx

```diff
@@ -1,5 +1,106 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { Form, Field } from "react-final-form";
+import Icon from "cozy-ui/react/Icon";
+import { Button } from "cozy-ui/react/Button";
+import IconEmail from "../../assets/icons/email.svg";
+import IconPhone from "../../assets/icons/phone-number.svg";
+import IconAddress from "../../assets/icons/location.svg";
+import IconCozy from "../../assets/icons/cozy.svg";
+import IconCompany from "../../assets/icons/company.svg";
+import IconBirthday from "../../assets/icons/calendar.svg";
+import IconNotes from "../../assets/icons/comment.svg";
 
-const ContactForm = () => <div>Contact Form </div>;
+const fields = [
+  {
+    labelKey: "firstname",
+    icon: null,
+    type: "text"
+  },
+  {
+    labelKey: "lastname",
+    icon: null,
+    type: "text"
+  },
+  {
+    labelKey: "phone",
+    icon: IconPhone,
+    type: "text"
+  },
+  {
+    labelKey: "email",
+    icon: IconEmail,
+    type: "text"
+  },
+  { labelKey: "address", icon: IconAddress, type: "text" },
+  { labelKey: "cozy", icon: IconCozy, type: "text" },
+  { labelKey: "company", icon: IconCompany, type: "text" },
+  { labelKey: "birthday", icon: IconBirthday, type: "text" },
+  { labelKey: "notes", icon: IconNotes, type: "text" }
+];
+
+const ContactFieldForm = ({ icon, label, type, input, meta }) => (
+  <div style={{ display: "flex" }}>
+    <div style={{ flex: "1" }}>
+      {icon && <Icon icon={icon} color="silver" />}
+      <label>{label}</label>
+    </div>
+    <input type={type} {...input} placeholder={label} />
```

Comment:
I failed to use the `cozy-ui/react/Input` component as it is not well controlled (rerendered and giving its value) by the wrapping form. But with a good :flashlight: and some concentration, I know we will succeed.